build/development-guide/deploy-smart-contracts/get-started/use-remix.md

```diff
@@ -42,18 +40,21 @@ contract BasicToken is ERC20 {
 Note: this is a simple ERC-20 contract based on the Open Zeppelin ERC-20 template. On construction, it creates the BasicToken with the symbol BAT, and mints the total initial supply.
 
 Below is the editor view.
+![](https://i.imgur.com/le9ZroU.png)
+
+Then click the `Solidity compiler` on the left sidebar sidebar, make sure that you use exactly same compiler version that is shown in above screenshot; and click the `Compile BasicToken.sol` button. 
 
-![](https://i.imgur.com/mdZhs7y.png)
+Remix will all of the Open Zeppelin dependencies and compile the contract.
```

Comment:
"Remix will include all of the Open Zeppelin dependencies and compile the contract."
missed a 'include' after 'will'